ProcessMaker/bpm
# redirect to login when global axios returns 401

closes #1600 

to test: 

change session.php 'lifetime' => 120 to 1. wait one minute and see that you will now be redirected to the login page. 

Please note that this also closes #1601 indirectly
